<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>9</y>
     <width>351</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Saisie</string>
    </attribute>
    <widget class="QPushButton" name="Button_SaisieAPSFS">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Saisie_FS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Saisie_APS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_FS">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_APS">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>APS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_EstOriente">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Orienté</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Algorithmes</string>
    </attribute>
    <widget class="QLabel" name="label_RetourPrufer">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>..................</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_Prufer">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>40</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Codage de Prufer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcul distance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Calcul_RacineDistance">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Inchange">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Racine</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_RetourDistance2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>..................</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom du fichier (txt) :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_nomFic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_sauvegarder_graphe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvegarder le graphe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_charger_graphe">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>420</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger un graphe</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOouNO">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>350</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>graphe orienté</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tarjan</string>
    </attribute>
    <widget class="QPushButton" name="Button_Tarjan">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Algorithme Tarjan</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="resultat_tarjan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetGraphe" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>891</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
